models/comments-model.js

```diff
@@ -0,0 +1,47 @@
+const db = require("../db/connection");
+const { selectArticles } = require("../models/articles-model");
+
+exports.selectComments = (articleId) => {
+  if (!Number.isInteger(Number(articleId))) {
```

Comment:
PSQL should be handling this, not a custom error handler.

Reply:
<img width="705" alt="Screenshot 2024-08-27 at 14 11 06" src="https://github.com/user-attachments/assets/d2cef604-f871-4789-a6f4-60e0c88fba55">